anndata/tests/test_backed_sparse.py

```diff
@@ -77,6 +77,24 @@ def callback(func, elem_name, elem, iospec):
     return csr_mem, csr_disk, csc_disk, dense_disk
 
 
+def test_empty_backed_indexing(
```

Comment:
Could this test also check for compatibility with an empty list of integers, e.g. `X[[], :]`?